Create WordsMatter.md (#2099)
betterscientificsoftware/bssw.io: CuratedContent/WordsMatter.md

## Words Matter for Inclusivity

<!--- deck text start --->
The ACM guide to inclusive language in computing emphasizes that inclusive language is crucial for promoting diversity and inclusion in the field.
<!--- deck text end --->

#### Contributed by [Rinku Gupta](https://github.com/rinkug)
#### Publication date: June 3, 2024

Resource information | Details
:--- | :---
Resource title | Words Matter
Authors | ACM Diversity and Inclusion Council
Website | [ACM - Words Matter](https://www.acm.org/diversity-inclusion/words-matter)

The ACM Diversity and Inclusion Council emphasizes the significant impact of language in the computing profession through their article [Words Matter](https://www.acm.org/diversity-inclusion/words-matter). 
This effort aims to replace offensive or exclusionary terminology with alternatives that are clearer and more inclusive. 
Examples of exclusionary words include "master/slave," "blacklist/whitelist," and "blind review". 

> The Council collaborates with major organizations like IBM, Apple, and Github in this endeavor. 

Reviewer: ```suggestion
The Council collaborates with major organizations like IBM, Apple, and Github. 
```

> They recommend using terms like "primary/secondary," "allowlist/blocklist," and "anonymous review" instead. 

Reviewer: ```suggestion
The article recommends alternatives such as "primary/secondary," "allowlist/blocklist," and "anonymous review", repsectively. 
```
